openapi/task_execution_service.openapi.yaml

```diff
@@ -741,6 +766,19 @@ components:
           example: 2020-10-02T10:00:00-05:00
           readOnly: true
       description: Task describes an instance of a task.
+    tesCallbackStatus:
+      type: object
+      required:
+      - task_id
+      - state
+      properties:
+        task_id:
```

Comment:
For consistency we should call this parameter (`task_id`) just `id` as it is called elsewhere.